api/File.json

```
@@ -224,7 +224,7 @@
               "version_added": "3.6"
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "≤68.10.1"
```

Both a suggestion here and a response to @foolip:

> can you advise on how to handle Firefox for Android for features that have likely been around since the very beginning (at least 5 years in this case) but where it's not possible to test on a real device?

Firefox has pretty good desktop-mobile parity. Based on the the rest of this API and a quick search in Bugzilla, I suspect that the Android features landed in the same versions (or nearest first release, for pre-version 4 stuff) as the desktop browser.

```suggestion
              "version_added": "4"
```